azure-migrate/migrate-at-scale-with-site-recovery/asr_startmigration.ps1

```diff
@@ -45,15 +47,19 @@ Function ProcessItemImpl($processor, $csvItem, $reportItem) {
     if ($protectableVM.ReplicationProtectedItemId -eq $null) {
         $processor.Logger.LogTrace("Starting protection for item '$($sourceMachineName)'")
         #Assumption storage are already created
-        $targetPostFailoverStorageAccount = Get-AzStorageAccount `
+        $targetPostFailoverStorageAccount = Get-AzureRmStorageAccount `
             -Name $targetPostFailoverStorageAccountName `
             -ResourceGroupName $targetStorageAccountRG
 
-        $targetResourceGroupObj = Get-AzResourceGroup -Name $targetPostFailoverResourceGroup
-        $targetVnetObj = Get-AzVirtualNetwork `
+        $targetPostFailoverLogStorageAccount = Get-AzureRmStorageAccount `
+            -Name $targetPostFailoverLogStorageAccountName `
+            -ResourceGroupName $targetLogStorageAccountRG
+
+        $targetResourceGroupObj = Get-AzureRmResourceGroup -Name $targetPostFailoverResourceGroup
+        $targetVnetObj = Get-AzureRmVirtualNetwork `
             -Name $targetPostFailoverVNET `
             -ResourceGroupName $targetVNETRG 
-        $targetPolicyMap  =  Get-AzRecoveryServicesAsrProtectionContainerMapping `
+        $targetPolicyMap  =  Get-AzureRmRecoveryServicesAsrProtectionContainerMapping `
```

Comment:
Do not use AzureRM; use the Az module. All documentation and sample scripts have moved to Az by the end of Q3.

Reply:
Thanks @sptramer for the review, have fixed the scripts to use Az module. Please take a look now. #sign-off